internal-payment-sdk/sdk/src/main/java/net/gini/android/internal/payment/review/reviewComponent/ReviewView.kt

```diff
@@ -217,17 +238,34 @@ class ReviewView(private val context: Context, attrs: AttributeSet?) :
     }
 
     private fun setDisabledIcons() {
-        setDisabledIcon(context.getString(R.string.gps_iban_hint), binding.ibanLayout)
-        setDisabledIcon(context.getString(R.string.gps_recipient_hint), binding.recipientLayout)
-        setDisabledIcon(context.getString(R.string.gps_purpose_hint), binding.purposeLayout)
+        if (reviewComponent?.reviewConfig?.editableFields?.contains(ReviewFields.IBAN) == false) {
+            setDisabledIcon(context.getString(R.string.gps_iban_hint), binding.ibanLayout)
+            binding.iban.isEnabled = false
+        }
+        if (reviewComponent?.reviewConfig?.editableFields?.contains(ReviewFields.RECIPIENT) == false) {
+            setDisabledIcon(context.getString(R.string.gps_recipient_hint), binding.recipientLayout)
+            binding.recipient.isEnabled = false
+        }
+        if (reviewComponent?.reviewConfig?.editableFields?.contains(ReviewFields.PURPOSE) == false) {
+            setDisabledIcon(context.getString(R.string.gps_purpose_hint), binding.purposeLayout)
+            binding.purpose.isEnabled = false
+        }
     }
 
     private fun handleInputFocusChange(hasFocus: Boolean, textInputLayout: TextInputLayout) {
         if (hasFocus) textInputLayout.hideErrorMessage() else textInputLayout.showErrorMessage()
     }
 
+//    fun clearFocus() {
```

Comment:
Commented code, do we need to delete it now?

Reply:
Thanks for catching that - deleted it.